builder.py

```diff
@@ -84,6 +87,7 @@ def build_cmake_extension(self, ext: CMakeExtension) -> None:
         cmake_args += [f"-DSYS_CONF_DIR={install_dir}"]
         cmake_args += [f"-DCORIOLIS_TOP={install_dir}"]
         cmake_args += [f"-DCORIOLIS_USER_TOP={install_dir}"]
+        cmake_args += [f"-DUSE_MANYLINUX=TRUE"]
```

Comment:
Builds on my machine :)
I'm just surprised by this line, as it seems that USE_MANYLINUX=TRUE is always set